src/lib.rs

```diff
@@ -777,12 +777,13 @@ fn pisa_to_ciff_from_paths(
 
 #[derive(Debug, Deserialize)]
 struct JsonDoc {
-    /// CIFF docid (must be an integer). In a typical CIFF index, this is 0..(num_docs-1).
-    id: i32,
+    /// Collection docid. CIFF will automatically assign an integer ID to this.
+    #[serde(default)]
+    id: String,
 
     /// Optional textual content for the document.
     #[serde(default)]
-    content: String,
+    _content: String,
```

Comment:
Can this be passed to CIFF or should we remove it? @amallia what was the intention behind passing it as `collection_docid` before?

Reply:
yes, this is correct! We should ignore content for now...